editors/code/tests/unit/index.ts

```diff
@@ -0,0 +1,38 @@
+import * as path from 'path';
+import Mocha from 'mocha';
+import glob from 'glob';
+
+export function run(): Promise<void> {
+    // Create the mocha test
+    const mocha = new Mocha({
+        ui: 'tdd',
+        color: true
+    });
+
+    const testsRoot = __dirname;
+
+    return new Promise((c, e) => {
```

Comment:
```suggestion
    return new Promise((resolve, reject) => {
```
What does `c` mean? Let's use the same names as here: https://github.com/rust-analyzer/rust-analyzer/blob/42dd0ce51ffee1dbfc58be7ebb0861aa2331b9cc/editors/code/src/net.ts#L124

Reply:
To tell the truth I don't know :) Copied the file from vscode-cpptools.
My guess is `Commit`, though I agree (resolve, reject)  looks better.